### DISCUSSION TITLE
deprecated resource?

### DISCUSSION THREAD

**Discussion by aniola** [microCOVID/microCOVID#1335] [Q&A]
Hiii

I was looking at your resources page (which was linked from the ? next to the mask options) and this section https://www.microcovid.org/paper/14-research-sources#person-risk says you're using COVID-19 Projections from https://covid19-projections.com/estimating-true-infections-revisited/ .  Which I clicked on just for fun, and ended up learning that it's **not being maintained** and hasn't been maintained for like a year **per https://youyanggu.com/blog/one-year-later .**  

Maybe this is something you know about and maybe it isn't.  I haven't actually been following the details.  In case it's useful, here are the resources the author suggests to replace it with: https://youyanggu.com/covid19-resources .  

Thanks for your patience if this isn't news and you've already adapted, in which case can you please update your research resources section?

I'm on the edge of my seat to know if microcovid is still using this resource.

Thanks again!

**Comment by kg6gfq** [comment]
Hopefully the maintainers will chime in with a definitive answer, but...

As I read it, MicroCovid is using the mathematical model developed by covid19-projections.com / Youyang Gu.  It isn't actually pulling data from (the now unmaintained) covid19-projections.com.  It sounds like covid19-projections.com shut down because it's data source (CTP) closed down in favor of data provided by other entities, so presumably the math is still valid.  

So I don't think this is an issue - MicroCovid isn't relying on data from covid19-projections.com, just using the math that Youyang Gu came up with.  

Of course, the maintainers of MicroCovid would know more, and I'd also be interested to hear from them!
